hudi-cli/src/test/java/org/apache/hudi/cli/integ/ITTestRepairsCommand.java

```diff
@@ -145,6 +166,60 @@ public void testDeduplicate() throws IOException {
     assertEquals(200, result.count());
   }
 
+  @Test
+  public void testDeduplicateWithUpdates() throws IOException {
```

Comment:
Can we rename the exists `testDeduplicate ` to `testDeduplicateWithDefault` or `testDeduplicateWithInserts`?